README.md

```diff
@@ -1,16 +1,20 @@
-[![AsyncAPI Banner and Logo](/public/img/logos/github-repobanner-website.png)](https://www.asyncapi.com)
+[![AsyncAPI Banner and Logo](./public/img/logos/github-repobanner-website.png)](https://www.asyncapi.com)
 
 ---
+
 <!-- ALL-CONTRIBUTORS-BADGE:START - Do not remove or modify this section -->
-[![All Contributors](https://img.shields.io/badge/all_contributors-28-orange.svg?style=flat-square)](#contributors-)
+
+[![All Contributors](https://img.shields.io/badge/all_contributors-22-orange.svg?style=flat-square)](#contributors-)
```

Comment:
Why are you changing this batch in README? As far I can see, there are 29 contributors added to the README.

Reply:
Thanks for pointing it out. it was a merge conflict that has been resolved.